### PR TITLE
[Hub Generated] Review request for Microsoft.Consumption to add version 2019-01-01

### PR DESCRIPTION
**Pull request by icodetolive**
If you are a MSFT employee you can view your [work branch via this link](https://portal.azure-devex-tools.com/app/branch/icodetolive/azure-rest-api-specs/dev-consumption-Microsoft.Consumption-2019-01-01).

### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/consumption/resource-manager/Microsoft.Consumption/stable/2019-01-01/consumption.json [hunk 1]
```diff
@@ -2799,7 +2799,7 @@
...
+      "description": "The scope associated with budget operations. This includes '/subscriptions/{subscriptionId}/' for subscription scope, '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}' for resourceGroup scope, '/providers/Microsoft.Billing/billingAccounts/{billingAccountId}' for Billing Account scope, '/providers/Microsoft.Billing/billingAccounts/{billingAccountId}/departments/{departmentId}' for Department scope, '/providers/Microsoft.Billing/billingAccounts/{billingAccountId}/enrollmentAccounts/{enrollmentAccountId}' for EnrollmentAccount scope, '/providers/Microsoft.Management/managementGroups/{managementGroupId}' for Management Group scope, '/providers/Microsoft.Billing/billingAccounts/{billingAccountId}/billingProfiles/{billingProfileId}' for billingProfile scope, 'providers/Microsoft.Billing/billingAccounts/{billingAccountId}/invoiceSections/{invoiceSectionId}' for invoiceSection scope..",
```
Comment (nschonni) [inline specification/consumption/resource-manager/Microsoft.Consumption/stable/2019-01-01/consumption.json]: Double period at the end

Comment (icodetolive) [reply specification/consumption/resource-manager/Microsoft.Consumption/stable/2019-01-01/consumption.json]: Removed it.

**Comment by icodetolive** [comment]
I request reviewers to review this pr.

**Review by sarangan12 — APPROVED** [review]

**Comment by jhendrixMSFT** [comment]
@AutorestCI regenerate azure-sdk-for-go

**Pull request merged**